kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -412,9 +422,14 @@ private Node createFieldEntry(PatternField patternField, int fieldNum) {
         commentIconRegion.getStyleClass().add("grey-comment-icon");
         outerHBox.getChildren().addAll(innerHBox, commentIconRegion);
         fieldVBoxContainer.getChildren().addAll(fieldLabel, fieldText, outerHBox);
+        fieldVBoxContainer.setOnMouseClicked(mouseEvent -> showContextMenuPopUp(patternField, fieldVBoxContainer, mouseEvent.getScreenX(), mouseEvent.getScreenY()));
         return fieldVBoxContainer;
     }
 
+    private void showContextMenuPopUp(PatternField patternField, Node node, double x, double y) {
+        contextMenu.show(node, x, y);
+    }
```

Comment:
Can this method be removed and simply calling `contextMenu.show(node, x, y);`?

Reasoning:

1. patternField isn't used.
2. It is just a one line pass through statement. Call it within the `.setOnMouseClicked(...)`
3. Less code. less indirection.

Reply:
This will be used in the later code to set the values that need to be removed or edited.